src/components/Translation/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 
 const Translation: React.FC<TranslationProps> = ({ trans }) => {
+  console.log(trans)
```

Comment:
Would you like to remove this `console.log`?

Reply:
Sorry, I forget to delete this log